Sources/Commands/PackageTools/InstalledPackages.swift

```diff
@@ -0,0 +1,130 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift open source project
+//
+// Copyright (c) 2023-2022 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See http://swift.org/LICENSE.txt for license information
+// See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import ArgumentParser
+import CoreCommands
+import TSCUtility
+import TSCBasic
+import PackageModel
+import Foundation
+
+extension SwiftPackageTool {
+    
+    struct Install: SwiftCommand {
+        static var configuration: CommandConfiguration {
+            CommandConfiguration(abstract: "Offers the ability to install executable targets of the current package.")
+        }
+        
+        @OptionGroup()
+        var globalOptions: GlobalOptions
+        
+        func run(_ tool: SwiftTool) throws {
+            let dotSwiftPMDir = try tool.fileSystem.getOrCreateDotSwiftPMDirectory()
+            
+            let env = ProcessInfo.processInfo.environment
+            
+
+            // ~/.swiftpm/bin/
+            let swiftpmBinDir = dotSwiftPMDir.appending(component: "bin")
+            
+            if let path = env.path, !path.contains(swiftpmBinDir.pathString), env["SWIFTPM_SHUT_UP_ABOUT_EXECUTABLE_PATH"] == nil {
+                tool.observabilityScope.emit(warning: "PATH doesn't include \(swiftpmBinDir.pathString)! This means you won't be able to access the installed executables by default, and will need to specify the full path. If you wish to stop seeing this warning, set the enviroment variable `SWIFTPM_SHUT_UP_ABOUT_EXECUTABLE_PATH`.")
+            }
```

Comment:
No need to introduce an additional env var here, this should already be covered by the `quiet` option in `LoggingOptions`, which this command already inherits via `GlobalOptions`.